Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -324,49 +308,112 @@ void InterfacePane::OnSaveConfig()
 }
 
-void InterfacePane::OnCursorVisibleNever()
+void InterfacePane::OnLanguageChanged()
 {
-  Settings::Instance().SetCursorVisibility(Config::ShowCursor::Never);
+  ModalMessageBox::information(
+      this, tr("Restart Required"),
+      tr("You must restart Dolphin in order for the change to take effect."));
 }
 
-void InterfacePane::OnCursorVisibleAlways()
+void InterfacePane::AddDescriptions()
 {
-  Settings::Instance().SetCursorVisibility(Config::ShowCursor::Constantly);
+  static constexpr char TR_TITLE_DATABASE_DESCRIPTION[] = QT_TR_NOOP(
+      "Uses Dolphin's database of properly formatted names in the Game List Title column."
+      "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static constexpr char TR_THEME_DESCRIPTION[] =
+      QT_TR_NOOP("Changes the appearance and color of Dolphin's buttons."
+                 "<br><br><dolphin_emphasis>If unsure, select Clean.</dolphin_emphasis>");
+  static constexpr char TR_TOP_WINDOW_DESCRIPTION[] =
+      QT_TR_NOOP("Forces the render window to stay on top of other windows and applications."
+                 "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static constexpr char TR_LANGUAGE_DESCRIPTION[] = QT_TR_NOOP(
+      "Sets the language displayed by Dolphin's User Interface."
+      "<br><br>Changes to this setting only take effect once Dolphin is restarted."
+      "<br><br><dolphin_emphasis>If unsure, select &lt;System Language&gt;.</dolphin_emphasis>");
+  static constexpr char TR_FOCUSED_HOTKEYS_DESCRIPTION[] =
+      QT_TR_NOOP("Requires the render window to be focused for hotkeys to take effect."
+                 "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static constexpr char TR_USE_COVERS_DESCRIPTION[] =
+      QT_TR_NOOP("Downloads full game covers from GameTDB.com to display in the Game List's Grid "
+                 "View. If this setting is unchecked the Game List displays a banner generated "
+                 "from the game's save files, and if the game has no save file displays a generic "
+                 "banner instead."
+                 "<br><br>List View will always use the save file banners."
+                 "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static constexpr char TR_DISABLE_SCREENSAVER_DESCRIPTION[] =
+      QT_TR_NOOP("Disables your screensaver while running a game."
+                 "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static constexpr char TR_CONFIRM_ON_STOP_DESCRIPTION[] =
+      QT_TR_NOOP("Prompts you to confirm you want to end emulation when you press the stop button "
+                 "or hotkey."
+                 "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static constexpr char TR_USE_PANIC_HANDLERS_DESCRIPTION[] =
+      QT_TR_NOOP("Prompts you to decide how to handle panic alerts."
```

Comment:
This one could be improved. I suggest:

> In the event of a error, Dolphin will halt to inform the user of the error and present choices on how to proceed.
> With this option disabled, Dolphin will "ignore" errors. Emulation will not be halted and the user will not be notified.

The primary use case for this setting in the past was errors that didn't break games could endlessly notify the user and prevent play. This exists as an option to allow users to get around this specific scenario. So my goal in the above is that if a user is stuck behind errors and doesn't know how to turn them off, the tooltip should inform them that this is what they want.

Also I'm specifically saying ignore here because that's the button you press to make the prompt go away in the above scenario.